### PR TITLE
Feature/upgrade dependencies

### PR DESCRIPTION
**Pull request by ruben-martin-lozano**
@davecarter @kikoruiz @aleanez @FranDelaCasa please review.

### CODE REVIEW

File: src/sui-pagination/index.jsx [hunk 1]
```diff
@@ -75,15 +72,15 @@ export default class Pagination extends React.Component {
 }
 
 Pagination.propTypes = {
-  pageNumber: React.PropTypes.number,
-  totalPages: React.PropTypes.number,
-  handlePaginate: React.PropTypes.func,
-  window: React.PropTypes.number
+  handlePaginate: React.PropTypes.func.isRequired,
+  nextText: React.PropTypes.string,
```
Comment (kikoruiz) [inline src/sui-pagination/index.jsx]: Is it OK to not to have text?

File: src/sui-pagination/index.jsx [hunk 2]
```diff
@@ -39,7 +36,7 @@ export default class Pagination extends React.Component {
           onClick={this.props.handlePaginate}
           data-pagenumber={this.props.pageNumber - 1}
           className='pagination-prev'>
```
Comment (carlosvillu) [inline src/sui-pagination/index.jsx]: This classname is OK ?!

**Comment by kikoruiz** [comment]
👍

File: src/sui-pagination/index.jsx [hunk 3]
```diff
@@ -52,7 +49,7 @@ export default class Pagination extends React.Component {
           onClick={this.props.handlePaginate}
           data-pagenumber={this.props.pageNumber + 1}
```
Comment (carlosvillu) [inline src/sui-pagination/index.jsx]: Really we need a data-* in a React Component ?!

Comment (ruben-martin-lozano) [inline src/sui-pagination/index.jsx]: not really, I havent refactored this code, but I can do it.

File: src/sui-pagination/index.jsx (hunk 2 above)
Comment (ruben-martin-lozano) [inline src/sui-pagination/index.jsx]: There's another PR where this code is refactored: https://github.com/SUI-Components/sui-pagination/commits/feature/addStyling. I'll see if I can merge both in this one.

**Comment by davecarter** [comment]
👍  perfect

File: docs/index.jsx [hunk 4]
```diff
@@ -1,12 +1,24 @@
...
 import './style.scss';
 import '../src/index.scss';
 
-React.render(
-  <Pagination
-    pageNumber={1}
-    totalPages={8}
-    windwo={3} />,
+const handlePaginate = (pageNumber, event) => {
+  console.log(`Go to page: ${pageNumber}`);
```
Comment (carlosvillu) [inline docs/index.jsx]: console!!!!!

File: src/sui-pagination/index.jsx [hunk 5]
```diff
@@ -1,14 +1,11 @@
...
-import i18n from '@schibstedspain/rosetta';
 
 export default class Pagination extends React.Component {
 
   getPaginationItems(){
-    const leftWindow = this.props.pageNumber - this.props.window;
-    const rightWindow = this.props.pageNumber + this.props.window;
+    const leftWindow = this.props.currentPage - this.props.window;
+    const rightWindow = this.props.currentPage + this.props.window;
     const min = leftWindow < 1
```
Comment (carlosvillu) [inline src/sui-pagination/index.jsx]: Puedes quitar los magic numbers ?!

Comment (ruben-martin-lozano) [inline src/sui-pagination/index.jsx]: Hecho! :)

**Comment by carlosvillu** [comment]
👍

**Pull request merged**